packages/node_modules/@node-red/nodes/core/storage/10-file.html

```diff
@@ -237,7 +237,8 @@
             }).text(label).appendTo(encSel);
             $("#node-input-filename").typedInput({
                 default: "msg",
-                types:[{ value: "str", label:"", icon:"red/images/typedInput/az.svg"}, "msg", "jsonata", "env"],
+                types:[{ value: "str", label: RED._("common.type.string"),
+                         icon: "red/images/typedInput/az.svg"}, "msg", "jsonata", "env"],
```

Comment:
There must be a reason why this `str` type is being redefined here without a label. My guess is we didn't want it to say `string` as it was where the user would enter a `file path`.... but that is a pretty weak reason.

It would be better to revert it to use the built-in `str` type rather than redefine it:

```suggestion
                types:["str", "msg", "jsonata", "env"],
```

It is probably better to be consistent here.

Reply:
I had forgotten the built-in `str` type. I changed the code.